tortoise/fields.py

```diff
@@ -29,11 +29,12 @@
 class _FieldMeta(type):
     def __new__(mcs, name, bases, attrs):
         if len(bases) > 1:
-            # Instantiate class with only the 1st base class (should be Field)
-            cls = type.__new__(mcs, name, (bases[0],), attrs)  # type: Type[Field]
-            # All other base classes are our meta types, we store them in class attributes
-            cls.type = bases[1] if len(bases) == 2 else bases[1:]
-            return cls
+            if bases[0] == Field:
```

Comment:
What aboud:
```py
if len(bases) > 1 and bases[0] is Field:
```
A single conditional statement is preferable if it is possible. Also when comparing types, `is` will ensure that first subclass is a field, and not looks like a field.

Reply:
Indeed, thanks :-)